modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerTest.java

```diff
@@ -341,49 +334,6 @@ public void testCreateTable() {
         assertSame(table, tblManagerFut.join().table(DYNAMIC_TABLE_NAME));
     }
 
-    /**
-     * Testing TableManager#writeTableAssignmentsToMetastore for 2 exceptional scenarios:
```

Comment:
Could you please also adjust `// We write this key only in HA mode. See TableManager.writeTableAssignmentsToMetastore.` in `ZoneRebalanceRaftGroupEventsListener`?